src/Posts/AddComment.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from "react";
+import ProfileIcon from "../Header/ProfileIcon";
+import "./AddComment.css"
+
+
+
+function AddComment (props) {
+
+    const { post, comments, setComments } = props
+
+    const [newComment, setNewComment] = useState({contactId: 1, postId: post.id, content: ""})
+
+    function resetForm() {
+        setNewComment(initial_state)
+    }
+
+    function handleSubmit(e) {
+        e.preventDefault()
+
+        const data = {
+            contactId: 1,
+            postId: post.id,
+            content: e.target[0].value
+        }
+
+        const userName = "LAVINIABENZAR"
+        const baseUrl = `https://boolean-api-server.fly.dev/${userName}`
```

Comment:
I would move `https://boolean-api-server.fly.dev/` to a single variable in its own file, maybe a `constants.js`, so you can import it where needed rather than repeating the same string in different places